src/components/context-selection/PeriodSelector.js

```diff
@@ -1,20 +1,195 @@
-import { SelectorBarItem } from '@dhis2/ui'
+import i18n from '@dhis2/d2-i18n'
+import { generateFixedPeriods } from '@dhis2/multi-calendar-dates'
+import {
+    Button,
+    InputField,
+    SelectorBarItem,
+    SingleSelectField,
+    SingleSelectOption,
+} from '@dhis2/ui'
 import PropTypes from 'prop-types'
 import React, { useState } from 'react'
+import styles from './PeriodSelector.module.css'
 
-export const PeriodSelector = ({ selectedPeriodInfo }) => {
+const ANNUAL_REPORT_PERIOD_TYPES = [
+    { label: i18n.t('Yearly'), id: 'YEARLY' },
+    { label: i18n.t('Financial Year (April)'), id: 'FYAPR' },
+    { label: i18n.t('Financial Year (July)'), id: 'FYJUL' },
+    { label: i18n.t('Financial Year (October)'), id: 'FYOCT' },
+    // todo: wait, what? should this be Nov? see /api/periodTypes
+    { label: i18n.t('Financial Year (December)'), id: 'FYDEC' },
+]
```

Comment:
I'm curious about these (copied here [kinda] from FixedPeriod.source.js):
1. Should we fetch these from the `/api/periodTypes` endpoint so we can get internationalized names from there?
2. In our test instances, I see Financial Years for April, July, October, and _November_, not December -- is that consistent across instances? 🤔

Reply:
1. I'm not sure api/periodTypes are internationalized? When I have my UI and database language set to French, I still get the names in English
2. There's more generally an issue with periodTypes not being in alignment with what is expected in our date generation. Mozafar can weigh in a lot more on this, but the period types from the api are not consistent, like it's `FinancialOct` but then `FinancialJuly`, and then I think as you point out some are missing. So I think the approach has been not to pull this from the api. Joseph, Mozafar, and I discussed this also last month.